.spelling

```diff
@@ -109,12 +110,13 @@ AssemblyScript
 Atlassian
 AttributeGen
 Auth0
+AuthPolicy
```

Comment:
Also out of order.

Reply:
Is there a possibility that `make lint` could check this?